examples/virtual-machines/windows/vm-joined-to-active-directory/example.auto.tfvars

```diff
@@ -0,0 +1,4 @@
+prefix         = "tf-ad-ex"
+location       = "westeurope"
+admin_username = "domadmin"
+admin_password = "Sup3r$ecRet_P@SsW0rd!"
```

Comment:
do we want to remove this file?

Reply:
I committed by mistake as I didn't realise *tfvars were not in the .gitignore, but then left it in place as an example of auto vars, happy to remove it though.